### ISSUE TITLE
Incomplete road network in Valhalla

### ISSUE THREAD

**Issue by coderon98** [valhalla/valhalla#5025]
Hi, there,
I tried to generate an isochrone for the pedestrian profile at 2m30s.  
```
{"locations": [{"lon": 2.50341044, "lat":48.817627256851594}], "action":"isochrone","contours":[{"color":"ff0000","time":2.5}], "costing":"pedestrian", "skip_opposites": false}
```
![image](https://github.com/user-attachments/assets/71fecb06-c5e2-4463-b4e5-73c640f33712)
As the form was inconsistent, I tried to see what “expansion” meant in order to analyze the road network underlying the isochrone calculation.
```
{"locations": [{"lon": 2.50341044, "lat":48.817627256851594}], "action":"isochrone","contours":[{"color":"ff0000","time":2.5}], "costing":"pedestrian", "skip_opposites": false}
```
![image](https://github.com/user-attachments/assets/2f928786-ddb8-43fc-801b-15178006df0f)

Many roads are not shown. This explains the poor quality of the isochrone calculation results (the roads omitted are perfectly practicable in reality).
I downloaded my data on December 05, 2024 and used valhalla version “3.5.1”.

Do you have a solution to this problem?

**Comment by coderon98** [comment]
Hi, I still haven't found a solution to my problem.
Can anyone help me?

**Comment by gknisely** [comment]
Not sure where you got your data from, but looking at data that I created last fall this seems fine.  Did you try converting you own data?

`http://localhost:8002/isochrone?json={%22locations%22%3A[{%22lat%22%3A48.818818021274986%2C%22lon%22%3A2.5004625320434575}]%2C%22costing%22%3A%22pedestrian%22%2C%22denoise%22%3A%220%22%2C%22generalize%22%3A%220%22%2C%22contours%22%3A[{%22time%22%3A2.5}]%2C%22polygons%22%3Afalse}`

![Image](https://github.com/user-attachments/assets/cf93da43-642b-44e3-9f19-8dd18d75b2a6)

**Comment by coderon98** [comment]
No, I haven't made any changes to the file.
I downloaded the data on December 02, 2023 from
https://planet.openstreetmap.org/

**Comment by gknisely** [comment]
I don't believe openstreetmap.org hosts any valhalla tiles for downloading.  You need to create your valhalla tiles locally.  This is a good tutorial.  https://gis-ops.com/valhalla-part-2-how-to-run-valhalla-on-ubuntu/

**Comment by coderon98** [comment]
@gknisely 
I downloaded the pbf data on this platform
(https://planet.openstreetmap.org/). Then I built the valhalla tiles locally (not the first time I've done this).

**Comment by gknisely** [comment]
Something looks to be incorrect with the data you created.  The streets that you highlighted in the second image are tagged with access=permissive.  I would try running some routes via http://valhalla.github.io/demos/routing/index-internal.html#loc=17,48.816942,2.504373  It looks like you have dropped and/or tossed edges in your data as I can route here just fine.  

![Image](https://github.com/user-attachments/assets/85c7e800-6e03-4c47-87f0-93bcfe485ce0)

**Comment by chrstnbwnkl** [comment]
Mh I can't seem to reproduce your initial example using our public API (https://valhalla1.openstreetmap.de/isochrone):

![Image](https://github.com/user-attachments/assets/831c4172-5491-42fd-bad8-bbc6e81c5adc)

At least the "patchy" parts I can't reproduce. Can you try getting latest OSM data and make sure you're using Valhalla built from latest master?

**Comment by coderon98** [comment]
Okay, I'll rebuild with the latest Openstreetmap data. The details I gave in my first post correspond to the bug reproduction situation (data update date, valhalla versions, ...).

**Comment by chrstnbwnkl** [comment]
Yeah sorry for not getting back to this right away... are there any other changes you made to the config? Maybe the default search radius in loki?

**Comment by coderon98** [comment]
No, I haven't made any changes to the configurations
